projects/storefrontlib/src/cms-components/product/product-intro/product-intro.component.html

```diff
@@ -4,13 +4,14 @@
       [rating]="product?.averageRating"
       [disabled]="true"
     ></cx-star-rating>
+
     <div class="count">({{ product?.numberOfReviews }})</div>
-    <a
-      class="btn-link"
-      *ngIf="reviewsTabAvailable | async"
-      (click)="showReviews()"
-      >{{ 'productSummary.showReviews' | cxTranslate }}</a
-    >
+
+    <ng-container *ngIf="reviewsTabAvailable | async">
```

Comment:
Test is failing at asserting `Show reviews` because the tab has not been rendered yet

Reply:
> Test is failing at asserting `Show reviews` because the tab has not been rendered yet

Reply:
this is actually working, but we have to change components' order in backoffice. will change the receipe soon, so this error will not show anymore